### ISSUE TITLE
Inconsistency in the format among CSS color functions

### ISSUE THREAD

**Issue by yarusome** [mdn/content#26434]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/hsl

### What specific section or headline is this issue about?

Syntax

### What information was incorrect, unhelpful, or incomplete?

Currently there're rather different formats for the "Syntax" section of CSS color functions.

For the contents immediately below the "Syntax" heading and above the "Values" heading:

  1. [`color()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color), [`color-contrast()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-contrast) (the name even deviates from [that](https://w3c.github.io/csswg-drafts/css-color-6/#colorcontrast) in the premature spec), [`color-mix()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix), [`hwb()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/hwb), [`lab()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/lab), [`lch()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/lch), [`oklab()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/oklab), and [`oklch()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/oklch) demonstrate the syntax with concrete values only.
  2. [`hsl()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/hsl) demonstrates various possible syntaxes with explaining comments *but* without concrete values in the code block, followed by an apparently redundent explanation on the syntax.
  3. [`rgb()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb) demonstrates various possible syntaxes with explaining comments *and* concrete values in the code block, followed by a explanation on the syntax.

`hsl()` is almost surely not doing the right thing here as this section usually starts with concrete examples, but I'm not sure which of the rest is better due to what comes in the "Values" subsection:

  1. `color()`, `color-contrast()`, `hwb()` start with the line "Functional notation: `notation`", followed by a definition list explaining the components in `notation`.
  2. `color-mix()`, `lab()`, `lch()`, `oklab()`, `oklch()` do the same, but with an apparently failed attempt to use definition lists.
  3. `hsl()` and `rgb()` explain the components already named above this subsection.

I'd like to open a PR to unify the format since it blocks the l10n of these pages, but I have to get informed on the preferred one.

### What did you expect to see?

A consistent format for this section across color functions.

**Comment by bsmth** [comment]
Thanks a lot for the detailed investigation @yarusome.

I think we should unify these like you propose with the same format used in [`lch()`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/lch) and others.

* Concrete values under syntax (not exhaustive) is fine if we're going into detail in the functional notation under 'values' with a breakdown of each value.
* Description lists for functional notation section look good.
* We do not need to go overboard with legacy syntax examples (commas between params and the `a` suffix for alpha channel) in the syntax code block. There's a note admonition on `rgb()` referencing `rgba()` and it's in plain English after the syntax code block about the comma separated values. This is better than showing a legacy example that can be copied and pasted, in my opinion.

````md
> **Note:** Legacy syntax is `...`

## Syntax

```css
function(1 2 3);
function(4deg 5% 6%);
```

### Values

Functional notation: `function(x y z [/ A])`

- `x`

  - : description

- `y`

  - : description

- `z`

  - : description

- `A`

  - : (usually) Alpha channel

The function also accepts a legacy syntax in which all values are separated with commas.
````

What do you think?

**Comment by bsmth** [comment]
Sounds great! You can go ahead with a PR for this if you like. Feel free to add me as a reviewer / assign it to me and I will take a look 👀

Thanks a lot :)
